dpd-client/tests/integration_tests/common.rs

```diff
@@ -780,6 +793,39 @@ impl Switch {
                 .join(", "))),
         }
     }
+
+    /// If no client_name is given, this returns the number of packets dropped
+    /// for a given reason.
+    ///
+    /// If a client_name is given, we look up the counter for that client
+    /// name and the given counter name.
+    ///
+    /// If that counter isn't in the set returned by dpd, we return
+    /// an error to the caller.
+    pub async fn get_counter(
```

Comment:
moved these functions in for use beyond the counters test (and having mcast tests all live in that module)